src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> lst;
```

Comment:
Perhaps a plural form such as tasks could be used to name the variable? Might it seem more intuitive that way?